src/components/TextWithIcon.tsx

```diff
@@ -0,0 +1,19 @@
+type Props = {
+  icon: string
+  text: string | undefined
+  alt: string
+}
+
+const TextWithIcon: React.FC<Props> = ({ icon, text, alt }) => {
+  return (
+    <div className="flex items-center mb-4 h-4 text-sm">
+      {text ? (
+        <>
+          <img className="h-full mr-2" src={icon} alt={alt} /> {text}
+        </>
+      ) : null}
+    </div>
```

Comment:
I had the idea that the positions of the different types of contact info would stay the same, regardless of which of them are missing. So it would look like:

Person 1|Person 2
--|--
+35831542462|
email@email.com|email@email.com
**@** Telegram|**@** Telegram

Instead of:
Person 1|Person 2
--|--
+35831542462|email@email.com
_email@email.com|**@** Telegram
**@** Telegram|

I think that might be clearer?

Reply:
We should bottom-align these, but I think gaps will look even weirder if there's things missing.

| Person 1 | Person 2 |
|-|-|
| +358401234567 | +358401234567 |
| email@email.com | |
| \@ telegram | \@ telegram |